Gemfile.lock

```diff
@@ -865,7 +865,7 @@ DEPENDENCIES
   web-console (~> 4.2)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 2.7.1p83
```

Comment:
Please revert the `Gemfile.lock` because we are not doing any changes to the Gemfile in this PR.